apache2/waf_logging/waf_format.proto

```diff
@@ -37,7 +37,6 @@ message Properties {
 message Waf_Format {
   optional string resourceId = 1;
   optional string operationName = 2;
-  optional string time = 3;
```

Comment:
the td-agent will add the time stamp for that, so we don't need to, might be milliseconds difference, but since we don't expose the milliseconds to jarvis, so it's fine, i have done the testing to verify that, also for the info/error/warning logs, they are using the timestamp td-agent added, so i think it's fine